packages/eas-cli/src/commands/update/configure.ts

```diff
@@ -219,7 +272,7 @@ async function configureAppJSONForEASUpdateAsync({
           'https://expo.fyi/eas-update-config.md'
         )}\n`
       );
-      Log.log(chalk.bold(JSON.stringify(newConfig, null, 2)));
+      Log.log(chalk.bold(JSON.stringify(newConfigOnlyAddedValues, null, 2)));
```

Comment:
is it possible for these values to be something other than updates.url && runtime version?

Reply:
Nope, see the construction of `newConfig` above each construction of `newConfigOnlyAddedValues`.